tools/update_repo.sh

```diff
@@ -558,6 +560,10 @@ EOF
         initiate_deb_metadata dists/$loop_dist/$component/binary-amd64/Packages
         for deb in $ws/$debdir/$loop_dist/$component/*/*/*.deb ; do
             [ -f $deb ] || continue
+            if [[ "$arch" != "x86_64" && \
+                  "$deb" != "_${arch}.deb" ]]; then
+              continue
+            fi
```

Comment:
1. I guess you had intention to use a regex match, but `"$deb" != "_${arch}.deb"` is exact match.
2. To be honest, I don't understand the logic here. Skip 'all' (architecture independent) packages? So whether arm64 users will have the tarantool-common package available? AFAIR, we have no the separate `binary-all` directory and 'all' packages now goes into the `binary-amd64` one. We should either introduce the `binary-all` directory or keep the current approach and deploy 'all' packages into both `binary-amd64` and `binary-arm64` (the latter should be easier to implement right now).

Reply:
I removed it.